cpp/ql/lib/semmle/code/cpp/ir/implementation/aliased_ssa/Instruction.qll

```diff
@@ -1856,12 +1856,12 @@ class InitializeDynamicAllocationInstruction extends SideEffectInstruction {
   }
 
   /**
-   * Gets the address of the allocation this instruction is initializing.
+   * Gets the operand of the allocation this instruction is initializing.
```

Comment:
Is this accurate, or should it be "Gets the operand of the address of the allocation this instruction is initializing."?

Reply:
Yeah, maybe yours is more accurate. The IR looks like this:
```
r1(void *)          = Call[operator new]              : func:r292_2, 0:r292_3
m1(unknown)   = ^CallSideEffect                 : ~m0
m2(unknown)   = Chi                             : total:m0, partial:m1
m3(unknown)   = ^InitializeDynamicAllocation    : &:r1
```
and the QLDoc attempts to describe the `&:r1` operand. It's the operand that represents the address that this instruction is initializing.